examples/virtualmachine.bicep

```diff
@@ -0,0 +1,27 @@
+param location string = resourceGroup().location
+param name string = 'vmtest1'
+
+resource vm 'Microsoft.Compute/virtualMachines@2022-08-01' = {
+  name: name
+  location: location
+  properties: {
+    hardwareProfile: {
+      vmSize: 'Standard_DS2_v2'
+    }
+    storageProfile: {
+      osDisk: {
+        createOption: 'FromImage'
+        managedDisk: {
+          storageAccountType: 'Standard_LRS'
+        }
+      }
+    }
+    // networkProfile: {
```

Comment:
Remove if not needed?

Reply:
Commented out as a TODO

Reply:
> Commented out as a TODO

Instead of commenting it out, please remove it. The portion that being removed is pretty trivial and leaving commented code in the code base is not a good practice.

If it truly is a TODO, then replace it with a TODO comment without code.